### DISCUSSION TITLE
Does Flipt server work with load balancers

### DISCUSSION THREAD

**Discussion by sgilhooly** [flipt-io/flipt#573] [Q&A] [answered]
Is it safe to run multiple replicas of a a Flipt server, behind a load balancer, pointing to the same database?

**Comment by markphelps** [comment, marked as answer]
Hi @sgilhooly, great question!

Yes, it should be safe to run multiple instances of Flipt behind a LB, assuming that you are **NOT using the [in-memory caching](https://flipt.io/docs/configuration/#caching)** as Flipt itself is stateless if in-memory caching is disabled (the default).

Also, I'm assuming you are using either MySQL or Postgres? I'd also recommend that you only [run migrations](https://flipt.io/docs/configuration/#migrations) from a single node as multiple nodes running migrations would likely cause errors.
